2025-10-07 17:26:18,161 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2025-10-07 17:26:19,906 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2025-10-07 17:26:22,919 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2025-10-07 17:26:24,767 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2025-10-07 17:26:25,187 | ERROR | OMAudit | user=hadoop | ip=10.9.0.17 | op=READ_VOLUME {"volume":"18890-volume-for-prepare"} | ret=FAILURE | 
org.apache.hadoop.ozone.om.exceptions.OMException: Volume 18890-volume-for-prepare is not found
	at org.apache.hadoop.ozone.om.VolumeManagerImpl.getVolumeInfo(VolumeManagerImpl.java:60)
	at org.apache.hadoop.ozone.om.OzoneManager.getVolumeInfo(OzoneManager.java:2800)
 
2025-10-07 17:26:26,584 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2025-10-07 17:26:27,070 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=LIST_VOLUMES {"prevKey":"","prefix":"","maxKeys":"1000","username":"hadoop"} | ret=SUCCESS |  
2025-10-07 17:26:27,264 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=LIST_VOLUMES {"prevKey":"55035-volume-for-prepare","prefix":"","maxKeys":"1000","username":"hadoop"} | ret=SUCCESS |  
